index.bs

```diff
@@ -4211,121 +4465,68 @@ Each {{WebRequestEvent}} has:
 </div>
 
 <div algorithm>
-  To <dfn>create a {{WebRequestResponseEventDetails}} object</dfn> given a
-  [=/request=] |request|, a [=/response=] |response|, a [=boolean=]
-  |requestsHeaders|, and a [=boolean=] |requestsAllHeaders|,
-  run the following steps:
-
-  1. Let |details| be the result of [=creating a WebRequestEventDetails object=]
-      given |request|.
+  To <dfn>create a {{WebRequestResponse}} object</dfn> given a [=/request=]
+  |request|, a [=/response=] |response|, and a {{RequestedHeaders}}
+  |requestedHeaders|, run the following steps:
 
-  1. Update the following fields in |details|:
+  1. Let |response| be a [=new=] {{WebRequestResponse}} with the following fields:
```

Comment:
Also initialize `redirectURL` to an empty `USVString`.

Reply:
For properties like this we should probably still say something like:

> redirectURL, which is a USVString or null, initially null.

Otherwise even if null is a valid value it's unclear what it is set to if none of the algorithms explicit set it.

Reply:
I updated all fields that aren't always set with ", initially null."